File: dart/lib/model/autoretrieve_init_body.dart
part of swagger.api;

class AutoretrieveInitBody {
  /* Autoretrieve's comma-separated list of addresses */
  String addresses = null;
/* Autoretrieve's public key */
  String pubKey = null;

  AutoretrieveInitBody();

  @override
  String toString() {
    return 'AutoretrieveInitBody[addresses=$addresses, pubKey=$pubKey, ]';
  }

  AutoretrieveInitBody.fromJson(Map<String, dynamic> json) {
    if (json == null) return;
    addresses = json['addresses'];
    pubKey = json['pubKey'];
  }

  Map<String, dynamic> toJson() {
    return {
      'addresses': addresses,
      'pubKey': pubKey
     };
  }

  static List<AutoretrieveInitBody> listFromJson(List<dynamic> json) {
    return json == null ? new List<AutoretrieveInitBody>() : json.map((value) => new AutoretrieveInitBody.fromJson(value)).toList();
  }

  static Map<String, AutoretrieveInitBody> mapFromJson(Map<String, Map<String, dynamic>> json) {
    var map = new Map<String, AutoretrieveInitBody>();
    if (json != null && json.length > 0) {
      json.forEach((String key, Map<String, dynamic> value) => map[key] = new AutoretrieveInitBody.fromJson(value));
    }
    return map;
  }
}

File: csharp-dotnet2/src/main/CsharpDotNet2/estuary-client/Model/AutoretrieveInitBody.cs
using System;
using System.Text;
using System.Collections;
using System.Collections.Generic;
using System.Runtime.Serialization;
using Newtonsoft.Json;

namespace IO.Swagger.Model {

  /// <summary>
  /// 
  /// </summary>
  [DataContract]
  public class AutoretrieveInitBody {
    /// <summary>
    /// Autoretrieve's comma-separated list of addresses
    /// </summary>
    /// <value>Autoretrieve's comma-separated list of addresses</value>
    [DataMember(Name="addresses", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "addresses")]
    public string Addresses { get; set; }

    /// <summary>
    /// Autoretrieve's public key
    /// </summary>
    /// <value>Autoretrieve's public key</value>
    [DataMember(Name="pubKey", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "pubKey")]
    public string PubKey { get; set; }


    /// <summary>
    /// Get the string presentation of the object
    /// </summary>
    /// <returns>String presentation of the object</returns>
    public override string ToString()  {
      var sb = new StringBuilder();
      sb.Append("class AutoretrieveInitBody {\n");
      sb.Append("  Addresses: ").Append(Addresses).Append("\n");
      sb.Append("  PubKey: ").Append(PubKey).Append("\n");
      sb.Append("}\n");
      return sb.ToString();
    }

    /// <summary>
    /// Get the JSON string presentation of the object
    /// </summary>
    /// <returns>JSON string presentation of the object</returns>
    public string ToJson() {
      return JsonConvert.SerializeObject(this, Formatting.Indented);
    }

}
}
